5-AdvancedScenarios/1-call-graph-bff/App/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "app",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "start": "node app.js",
+        "preinstall": "cd ./client && npm install",
+        "build": "cd ./client && npm run build"
```

Comment:
why not have a prestart script instead of build?

Reply:
Updated!